pkg/beacon/writer.go

```diff
@@ -56,7 +56,7 @@ func (w *writer) start() {
 
 				if err != nil ||
 					((info.Updated == 0) && (info.UpsertedId == nil)) {
-					log.WithFields(log.Fields{
+					w.log.WithFields(log.Fields{
```

Comment:
There's a whole bunch of these types of changes. This was causing the messages to be printed out to stdout instead of going to the log file only.